src/Notification/ImportantNotification.js

```diff
@@ -0,0 +1,117 @@
+/* @flow */
+
+import * as React from 'react';
+import { Animated, View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { Icon } from '../Icon';
+import Touchable from '../Button/Touchable';
+import StyleSheet from '../PlatformStyleSheet';
+
+import type { StylePropType } from '../PlatformStyleSheet/StyleTypes';
+import type { NotificationStyleType } from '../types';
+
+import Text from '../Text';
+
+type Props = {|
+  +style?: StylePropType,
+  +notificationStyle: NotificationStyleType,
+  +warningTitle: React.Node | string,
+  +warningMessage: React.Node | string,
+  +onPress: () => void,
+|};
+
+const AnimatedTouchable = Animated.createAnimatedComponent(Touchable);
+
+export default function ImportantNotification({
+  style,
+  onPress,
+  warningTitle,
```

Comment:
I believe these `warning` prefixes are now artefacts of the preview naming convention. Could you update them, pretty please?